crates/transaction-pool/src/blobstore/disk.rs

```diff
@@ -329,11 +343,11 @@ impl DiskFileBlobStoreInner {
             }
         }
         if cache_miss.is_empty() {
-            return Ok(res)
+            return Ok(res);
```

Comment:
please no trailing semicolons, for some reason on mac they're added, idk why...

Reply:
Weird, I checked the rustfmt and saw that they're disabled, yet had them pop up in my push too! Removed them.

Reply:
For everyone who has a similar format problem,  you need to use cargo +nightly to format the code, or you can run `make fmt` before push the commits.

Reply:
unfortunately that does not solve it on mac for some of us :(

Reply:
that's wired, maybe the `nightly` version on mac is different